common/controllers/form-wizard/index.js

```diff
@@ -135,20 +135,7 @@ class BaseController extends Controller {
   getErrors(req, res) {
     const errors = super.getErrors(req, res)
     const { fields } = req.form.options
-    const errorList = map(errors, error => {
-      return {
-        html: fieldHelpers.getFieldErrorMessage({
-          ...fields[error.key],
-          ...error,
-        }),
-        href: `#${fields[error.key].id}`,
-      }
-    })
-
-    return {
-      ...errors,
-      errorList,
-    }
+    return fieldHelpers.addErrorListToErrors(errors, fields)
```

Comment:
Is this change being used anywhere still? It looks like it was put in place for a different approach that you then changed.

Maybe we can even drop this commit/change until we have a need for it?

Reply:
I actually think it makes the tests for the form-wizard controller cleaner, so personally I think it should stay for that reason alone, in addition to making it reusable

Reply:
That’s fair.